FAQs/Projects.md

```diff
@@ -11,6 +11,38 @@ FAQs for projects includes questions about project information.
 Team admin or Publisher can regenerate cert for Publishing projects.
 Team admin/Publisher/Subscriber can regenerate cert for Subscribing projects.
 
+## How to encrypt MQTT payloads?
 
+Encrypted value of JWE is represented as the concatenation of the Base64 encoded JWE Protected Header, JWE Encrypted Key, JWE Initialization Vector, JWE Ciphertext, and JWE Authentication Tag separated by “.”
+
+BASE64URL(UTF8(JWE Protected Header)) || '.' ||
+
+BASE64URL(JWE Encrypted Key) || '.' ||
+
+BASE64URL(JWE Initialization Vector) || '.' ||
+
+BASE64URL(JWE Ciphertext) || '.' ||
+
+BASE64URL(JWE Authentication Tag)
+
+*For **end to end encryption of MQTT payload**, the payload should be **encrypted on the publisher side** and **decrypted on the subscriber side***
+*Options* are as follows:
+1. Publisher to generate their own encryption keys and manually passing it to subscribers.
+2. We can generate the encryption keys and put it in the topic details page (something similar to project connection page for certs).
```

Comment:
remove no.2. No.2 is more for internal info.

Reply:
For end-to-end encryption of ... subscriber side, publisher to generate their own ... to subscribers.